style.css

```diff
@@ -46,28 +48,28 @@ html, body {
     float: right;
     z-index: 4;
```

Comment:
If you use z-indexes, it's common to use bigger numbers. That will save you some headache in case you have an element in the future that has to be something in between z-index 4 and z-index 5, for example.

Reply:
Ok, noted it